text/0129-declarative-shadow-dom-polyfill.md

```diff
@@ -1,6 +1,6 @@
 ---
 title: Declarative ShadowDOM polyfill
-status: APPROVED
+status: IMPLEMENTED
```

Comment:
[Not actually implemented yet](https://gus.lightning.force.com/lightning/r/ADM_Work__c/a07EE00001RkJc4YAF/view)